nodes/src/nodes/telegram/IEndpoint.py

```
@@ -0,0 +1,410 @@
+            debug(f'Telegram: file_path={file_path}, file_size={file_size}')
+            download_url = f'https://api.telegram.org/file/bot{self._bot_token}/{file_path}'
+            async with self._http_session.get(download_url) as resp:
+                content = await resp.read()
+                if len(content) == 0:
+                    debug(f'Telegram: WARNING — 0 bytes downloaded! headers={dict(resp.headers)}')
+                return content
+        except Exception as e:
+            debug(f'Telegram: download error for {file_id}: {e}')
+            return None
+
+    async def _send_message(self, chat_id: int, text: str):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_send_message`

Add return type annotation: `None`

(ANN202)